extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -160,6 +161,24 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Prefetches and preconnects URLs related to the analytics.
+   * @param {boolean=} opt_onLayout
+   * @override
+   */
+  preconnectCallback(opt_onLayout) {
+    let url;
+    if ((getMode().localDev || getMode().test) &&
```

Comment:
Can't this just call getLibScriptUrl to remove duplicate code?

Reply:
Done.
I thought about it, but not sure if I should do so because we need to export that function.